src/components/Dashboard/ANTLockedDistribution.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react'
 import { GU, Link, textStyle, useTheme } from '@aragon/ui'
 
-import { useCourtConfig } from '../../providers/CourtConfig'
 import { formatUnits } from '../../lib/math-utils'
+import { getANTToken } from '../../utils/known-tokens'
 
-function ANJLockedDistribution({ distribution, text }) {
+function ANTLockedDistribution({ distribution, text }) {
   const theme = useTheme()
-  const {
-    anjToken: { decimals, symbol },
-  } = useCourtConfig()
+  const antToken = getANTToken()
```

Comment:
As noted already, let's keep using the courtConfig on all this cases.
```suggestion
  const { token: antToken } = useCourtConfig()
```